From 2bbc67cddc5f5bcba6a3a1cd980b54290d50cc03 Mon Sep 17 00:00:00 2001
From: Jiri Cerny <ji.cerny@gmail.com>
Date: Wed, 5 Oct 2016 21:03:26 +0200
Subject: shell-search-provider: do not search for whole words only

Starting from 3.20 the shell search works only for whole words, not
prefixes. Make it work again.

https://bugzilla.gnome.org/show_bug.cgi?id=764587
---
 libnautilus-private/nautilus-search-engine-tracker.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/libnautilus-private/nautilus-search-engine-tracker.c
+++ b/libnautilus-private/nautilus-search-engine-tracker.c
@@ -304,7 +304,7 @@ nautilus_search_engine_tracker_start (Na
                                "  nfo:fileLastAccessed ?atime;"
 			       "  tracker:available true;");
 
-        g_string_append_printf (sparql, " fts:match '\"%s*\"'", search_text);
+        g_string_append_printf (sparql, " fts:match '\"%s\"*'", search_text);
 
 	if (mime_count > 0) {
 		g_string_append (sparql, "; nie:mimeType ?mime");
